tests/publiccloud/download_repos.pm

```diff
@@ -55,12 +56,22 @@ sub run {
         assert_script_run('touch /tmp/repos.list.txt');
 
         my $ret = 0;
+        my $reject = "'robots.txt,*.ico,*.png,*.gif,*.css,*.js,*.htm*'";
+        my $regex = "'s390x\\/|ppc64le\\/|kernel*debuginfo*.rpm|src\\/'";
+        my $incident;
         for my $maintrepo (@repos) {
-            next if $maintrepo !~ m/^http/;
+            $incident = $1 while $maintrepo =~ /\/Maintenance:\/(\d+)/g;
+            die "We did not detect incident number for URL \"$maintrepo\". We detected \"$incident\"" unless $incident =~ /\d+/;
+            script_retry("curl -s https://build.suse.de/attribs/SUSE:Maintenance:$incident -o /tmp/$incident.txt");
```

Comment:
I think it would be a good idea to add a `-f` here so `curl` can return a failed retval in case the URL does not return what we expect:
```
# curl -s mango.qa.suse.de/non-existent -o non-existent ; echo $?
0
# md5sum non-existent ; rm non-existent
ddecdfa0919ead549d583b866effceb0  non-existent
# curl -s -f mango.qa.suse.de/non-existent -o non-existent ; echo $?
22
# md5sum non-existent ; rm non-existent
md5sum: non-existent: No such file or directory
rm: cannot remove 'non-existent': No such file or directory
```
Specially important since L66 is assuming "non embargoed" as the default, and only skipping it if the file downloaded by curl matches with `OBS:EmbargoDate`.

Reply:
Wow, how come I did know this before? Thank you!